resources/profiles/Creality/process/0.24mm Draft @Creality K1Max (0.4 nozzle).json

```diff
@@ -12,10 +12,10 @@
     "bottom_shell_layers": "3",
     "bottom_shell_thickness": "0",
     "bridge_flow": "1",
-    "bridge_speed": "50",
+    "bridge_speed": "25",
+    "internal_bridge_speed" : "70",
     "brim_width": "5",
     "brim_object_gap": "0.1",
-    "compatible_printers": [],
```

Comment:
Is the default already `[]` and this is redundant, or does deleting it set an explicit `null` / ignore? I think the OP just pulled it from the sample:
https://github.com/SoftFever/OrcaSlicer/blob/e70da30753933f832d4f15dd36b8bc498b26e563/resources/profiles_template/Template/process/process%20template.json#L108